.github/workflows/test-suite-desktop-e2e-pr.yml

```
@@ -39,6 +39,29 @@ jobs:
         with:
           project: "desktop"
 
+  post-currents-link:
```

actionlint (1.7.4)

job "post-currents-link" needs job "build-desktop" which does not exist in this workflow

(job-needs)